server/src/main/java/org/apache/iotdb/db/engine/memtable/AbstractMemTable.java

```diff
@@ -129,6 +136,12 @@ private IWritableMemChunkGroup createAlignedMemChunkGroupIfNotExistAndGet(
 
   @Override
   public void insert(InsertRowPlan insertRowPlan) {
+    // if this insert plan isn't from storage engine, we should set a temp device id for it
```

Comment:
```suggestion
    // if this insert plan isn't from storage engine(currently may be from wal), we should set a temp device id for it
```

Reply:
Wal is also from storage engine. Currently is from test.